test/nbrowser/ChoiceList.ts

```diff
@@ -447,7 +447,7 @@ describe('ChoiceList', function() {
 
   it('should allow reasonable conversions between ChoiceList and other types', async function() {
     await gu.enterGridRows({rowNum: 1, col: 'A'},
-      [['Hello'], ['World'], [' Foo,Bar;Baz!,']]);
+      [['Hello'], ['World'], [' Foo,Bar;Baz!,"Qux",']]);
```

Comment:
I suggest replacing "Qux" with a stranger value, like one that includes a comma and maybe leading or trailing whitespace, to ensure that such values are handled without surprises or errors.